.github/workflows/contrib-tests.yml

```diff
@@ -22,7 +22,17 @@ permissions:
   # contents: read
   # deployments: read
 jobs:
+  DeployPostgres:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install PostgreSQL
+        run: |
+          sudo apt install postgresql -y
+      - name: Start PostgreSQL service
+        run: sudo service postgresql start
```

Comment:
I set it up this way because when matrix.os is windows-2019 or macos-latest, I believe it will not be able to apt install postgresql. Please let me know if I am mistaken.